Automattic/wp-calypso
# [Jetpack Agency] Add contact person and company website fields

## This should not be merged before D86484-code has been deployed.

### Proposed Changes

Add two new fields that will be displayed when a partner signs up or when they update their company details (contact person and company website) that are intended to help the business team.

---

### Context

* 1201801459945917-as-1202339487554264
* pdpAdu-E6-p2
* D86484-code

---

### Testing Instructions



* Spin up this branch
* Apply D86484-code to your sandbox if it hasn't been deployed yet.
  * Remember to change your DNS if you have to sandbox the API. 
* Go to http://jetpack.cloud.localhost:3000/agency/signup in an incognito window
  * You either have to log in with a WPCOM account that doesn't have a partner associated to it yet, or you'll have to create a new account.
* Go to http://jetpack.cloud.localhost:3000/partner-portal/company-details and verify all your data shows up
* Change the `Contact person` and `Company website` values and submit the details form.

_Note: you can also modify the "Create Partner" endpoint to update your existing partner instead of creating a new one and remove the redirect condition for the signup form if you don't want to create a new WPCOM account, but it's quite a bit of finessing._

---

### Screenshots

#### Sign up

![Screenshot 2022-08-25 at 13 20 40](https://user-images.githubusercontent.com/3846700/186653949-88c1cf57-e3d6-4421-9400-d3687aa8f5c8.jpg)

#### Update company details

![Screenshot 2022-08-25 at 13 18 15](https://user-images.githubusercontent.com/3846700/186653977-399100ed-1ec0-437b-bec5-262b9cb090ba.jpg)

---

### Pre-merge Checklist
